pkgs/development/python-modules/pyvirtualdisplay/default.nix

```
@@ -0,0 +1,25 @@
+{ stdenv, buildPythonPackage, fetchPypi, EasyProcess, entrypoint2, pathpy
+, pillow, pyscreenshot, nose
+}:
+
+buildPythonPackage rec {
+  pname = "PyVirtualDisplay";
+  version = "0.2.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1ankzn4d3akm3dkbsl1w988kjgad07lp3brh1kkwn9v75pm073b5";
+  };
+
+  propagatedBuildInputs = [ EasyProcess ];
+  checkInputs = [ entrypoint2 nose pathpy pillow pyscreenshot ];
+
+  checkPhase = "nosetests -v .";
```

```suggestion
  checkPhase = ''
    runPhase preCheck
    nosetests -v .
    runPhase postCheck
  '';
```

So that we don't lose the pre and post phases :)